api-extractor/src/DocElementParser.ts

```diff
@@ -28,6 +28,29 @@ export default class DocElementParser {
    */
   private static _hrefRegEx: RegExp = /^[a-z]+:\/\//;
 
+  public static getAsText(collection: IDocElement[], reportError: (message: string) => void): string {
+    let text: string = '';
+
+    collection.forEach(docElement => {
+      switch (docElement.kind) {
+        case 'textDocElement':
+          text += `${(docElement as ITextElement).value} `;
+          break;
+        case 'linkDocElement':
+          // links don't count towards the summary
+          break;
+        case 'seeDocElement':
+          // see doesn't count towards the summary
+          break;
+        default:
+          reportError('Unexpected item in JsDoc collection');
```

Comment:
Really this is the IDocElement collection.

It would be easy for devs to overlook this requirement, causing an annoying bug for people.  I would suggest to add a note to the "kind" enum reminding people to update this function when they add new kinds.